proto/v1/message.proto

```diff
@@ -54,3 +54,48 @@ message Sources {
     repeated string multiaddrs = 2;
   }
 }
+
+
+message BidBotEvent {
+  google.protobuf.Timestamp ts = 1;
+  oneof type {
+    StartUp start_up = 2;
+    StartFetching start_fetching = 3;
+    ErrorFetching error_fetching = 4;
+    StartImporting start_importing = 5;
+    EndImporting end_importing = 6;
+    Finalized finalized = 7;
+    Errored errored = 8;
+  }
+  message StartUp {
+    string semantic_version = 1;
+    string storage_provider_id = 2;
+    uint64 deal_start_window = 3;
+    bool cid_gravity_configured = 4;
```

Comment:
a few flags we care about most. anything missing?

Reply:
Moving `storage_provider_id` to the top level `BidBotEvent` may make life easier in any way for auctioneer?

Reply:
Yeah good point. I'll see if auctioneer need this field for storing data or logging when implementing this.

Reply:
Yea good point @jsign. I was also wondering how auctioneer would know who is sending these events? Is there any auth mechanism? If we already are using libp2p and we know who the connected bitbots are there, would it be better to leverage that channel instead of grpc?

Reply:
Oh, or are these protos going to be sent over libp2p? Maybe I just answered my own question.

Reply:
Yes they are sent over libp2p.